### PR TITLE
ci: make clean-test-output a script for reuse

### PR DESCRIPTION
**Pull request by weihanglo**
### What does this PR try to resolve?

Just make CI workflow yaml a bit clear by reusing a script.

### How should we test and review this PR?

Let CI pass.

### CODE REVIEW

File: ci/clean-test-output.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# This script remove test and benchmark output and displays disk usage.
```
Comment (epage) [inline ci/clean-test-output.sh:2]: Minor thing but should this live in the workflows directory?  Thats where I tend to put stuff like this which also helps keep the root clean

Comment (weihanglo) [reply ci/clean-test-output.sh:2]: Sounds good to me. Do you want me moving them all in this PR?

Comment (weihanglo) [reply ci/clean-test-output.sh:2]: BTW rust-lang/rust puts them under `src/ci`.

Comment (weihanglo) [reply ci/clean-test-output.sh:2]: BTW rust-lang/rust puts them under `src/ci`. I am okay with all these options.

**Pull request merged**